docs/howtos/run_sql.md

```diff
@@ -0,0 +1,10 @@
+## Loading Data to a Source
```

Comment:
can you explain what is db_path and sql_path? You have only 2 sentences here, try to expand a bit the documentation for the `SQLtoDF` task